src/style.css

```diff
@@ -1,3 +1,92 @@
+:root {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-size: 12px;
+  --bg-color: #fff;
+  --a-color: #000;
+  --a-hover-color: #00a0e3;
+  margin: 0;
+  padding: 0;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url("Roboto/Roboto-Medium.ttf");
+}
+
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
+  color: black;
+}
+
+a {
+  padding: 22.5px;
+}
```

Comment:
you want to add padding to `nav__link` that is a specific `<a>` tag, you don't want to add the padding to all of the `<a>` tags on your page. Also, move these styles to other `nav__link` styles.